src/magic_folder/test/test_web.py

```diff
@@ -118,18 +121,16 @@ class StatusTests(AsyncTestCase):
     """
     Tests for ``magic_folder.status.status``.
     """
-    @given(folder_names(), absolute_paths().map(FilePath))
+    @given(folder_names(), absolute_paths_utf8().map(FilePath))
```

Comment:
ooh, that's nice. Didn't know we can call other functions inside `@given`. Nice!